test/docgen/view/documentation.test.ts

```diff
@@ -1,48 +1,45 @@
-import * as reflect from 'jsii-reflect';
 import { Documentation } from '../../../src';
-import { Assemblies } from '../assemblies';
 
-const assembly: reflect.Assembly = Assemblies.instance.withoutSubmodules;
-const assemblyWithSubmodules: reflect.Assembly = Assemblies.instance.withSubmodules;
+const ASSEMBLIES = `${__dirname}/../../__fixtures__/assemblies`;
+
+// this is a little concerning...we should be mindful
+// if need to keep increasing this.
+jest.setTimeout(2 * 60 * 1000);
```

Comment:
My experience is that this can be avoided by reducing `jest` concurrency (which by default is somewhat unbounded, and causes excessive IO-queuing, especially in CI environments with constrained resources).

Reply:
Ok, good to know - we'll keep an eye on this.